ajax/libs/jquery-rss/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "jquery-rss",
+  "filename": "jquery.rss.min.js",
+  "description": "An easy-to-use rss plugin for jquery with templating.",
+  "version": "3.3.0",
+  "keywords": [
+    "rss",
+    "jquery",
+    "ender"
+  ],
+  "author": "Sascha Depold <sascha@depold.com> (http://depold.com)",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/sdepold/jquery-rss.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/sdepold/jquery-rss.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "*.(js|map)"
```

Comment:
This auto-update config can't get any file.
Also, we use excluded mode when `dist` as basePath.
Let's modify as the followings:
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/sdepold/jquery-rss.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/!(*.jar)"
        ]
      },
      {
        "basePath": "src",
        "files": [
          "jquery.rss*"
        ]
      }
    ]
  }
```

Thanks.